index.html

```diff
@@ -14,11 +15,11 @@
       <h2>The Main Content</h2>
       <div class='left'>
 	This is the left side
-	<img src='https://jesse.sh/img/me.jpg'>
+	<img src="https://jesse.sh/img/me.jpg"/>
```

Comment:
👍 looking for `alt` but will count this (but keep in mind `/` doesn't have an effect, alt is important for screen readers and failures to load assets)